### PR TITLE
Atomize the repo

### PR DESCRIPTION
**Pull request by Evankhell561**
atomize the repo.
makes a lot of changes to the way the code is written. this should not be noticeable at all.

### CODE REVIEW

File: code/__HELPERS/unsorted.dm [hunk 1]
```diff
@@ -1049,7 +1049,7 @@ proc/is_hot(obj/item/W)
 	if(W.sharp) return TRUE
 	return ( \
 		W.sharp														|| \
-		istype(W, /obj/item/weapon/tool)							|| \
+		istool(W)													|| \
```
Comment (Kurgis) [inline code/__HELPERS/unsorted.dm:1052]: Was this meant to keep its tabs?

File: code/modules/hivemind/self_defense.dm [hunk 2]
```diff
@@ -134,7 +134,7 @@
 	for(var/i = 1 to amount)
 		var/turf/spawn_loc = pick(places_to_spawn)
 		champion = new champion_path(spawn_loc)
-		champion.maxHealth += 150
+		champion.adjustMaxHealth(150)
```
Comment (TorinoFermic) [inline code/modules/hivemind/self_defense.dm:137]: You kinda used wrong function : setMaxHealth should be used, the next line can use a function adjustHealth

Comment (TorinoFermic) [reply code/modules/hivemind/self_defense.dm:137]: But reading more this function, why we have adjust**Max**Health and setMaxHealth

Comment (TorinoFermic) [reply code/modules/hivemind/self_defense.dm:137]: Okay, nevermind I found your newly added function. It turns out adjustMaxHealth hasnt never existed.

**Review by TorinoFermic — APPROVED** [review]

**Comment by TheShown911** [comment]
Clean

**Pull request merged**